src/components/base/Block.tsx

```diff
@@ -25,6 +25,7 @@ export interface Props {
   ['data-testid']?: string
   description?: ReactNode
   actions?: Action[]
+  moreClass?: string[]
```

Comment:
on appelle cette variable className dans tous nos composants
```suggestion
  className?: string
```